specification/automation/resource-manager/Microsoft.Automation/stable/2015-10-31/dscCompilationJob.json

```diff
@@ -203,6 +213,63 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Automation/automationAccounts/{automationAccountName}/compilationjobs/{jobId}/streams/": {
```

Comment:
Is this adding a new API or documenting an existing one that wasn't listed before?

Reply:
Documenting existing API which wasn't listed earlier.